cocos/ui/UIWidget.cpp

```diff
@@ -783,7 +783,9 @@ void Widget::propagateTouchEvent(cocos2d::ui::Widget::TouchEventType event, coco
     Widget* widgetParent = getWidgetParent();
     if (widgetParent)
     {
+        widgetParent->_hittedByCamera = _hittedByCamera;
```

Comment:
_hittedByCamera is a temp variable in touch begin->move->end event period, don't need save at all.